src/lib/aggregations/get-bucket-selector.ts

```diff
@@ -0,0 +1,15 @@
+export type BucketSelectorAggregation = {
+    bucket_selector: BucketSelectorAggregationBody
+}
+
+export type BucketSelectorAggregationBody = {
+    buckets_path: Record<string, string>
+    script: string
+}
+
+export const getBucketSelectorAggregation = (script: string, bucketsPath: Record<string, string>): BucketSelectorAggregation => ({
```

Comment:
`bucketPath` to `bucketPaths`?

Reply:
hmm, maybe we should stick to same names as in Elastic itself?